Sources/UID2/Networking/DataEnvelope.swift

```diff
@@ -44,6 +44,8 @@ internal enum DataEnvelope {
 extension Data {
     /// A convenience initializer for converting from a Data representation of a base64 encoded string to its decoded Data.
     init?(base64EncodedData: Data, options: Data.Base64DecodingOptions = []) {
+        // https://github.com/realm/SwiftLint/issues/5263#issuecomment-2115182747
+        // swiftlint:disable:next non_optional_string_data_conversion
```

Comment:
Nitpick: `swiftlint:disable:next non_optional_string_data_conversion` what does this mean, do we need to add more explanation here

Reply:
nvm. I am new to the swift. It is used for SwiftLint to disable non_optional_string_data_conversion rule for the next line of code temporarily.